install/kubernetes/cilium/values.yaml

```diff
@@ -758,6 +758,10 @@ certgen:
   ttlSecondsAfterFinished: 1800
   # -- Labels to be added to hubble-certgen pods
   podLabels: {}
+  # -- Annotations to be added to the hubble-certgen initial Job and CronJob
+  annotations:
```

Comment:
seems like we generally do this with either something like:

```yaml
cilium:
    annotations: {}
```
or

```yaml
podAnnotations: {}
```

Perhaps it would be more consistent to do something like:

```
...
    job:
        annotations: {}
    cronJob:
        annotations: {}
```

for consistency?

Reply:
Thanks for the feedback @tommyp1ckles 

I actually made it like this in the hope that it would be consistent, or at least similar look & feel.

modeled after `podAnnotations: {}` and `podLabels: {}` it would most naturally be

```yaml
jobAnnotations: {}
cronJobAnnotations: {}
```

however.. adding more and more single-level keys like this isn't a great solution. So I opted for nesting it under one annotations keys in order to make it more future proof. I.e. it could eventually be:

```yaml
annotations:
    all: {}
    job: {}
    cronjob: {}
    pod: {}
labels: 
    all: {}
    pod: {} # theoretically podLabels could be moved here 
    job: {}
    cronJob: {}
```

this makes the object easily extensible, whilst the inverse approach (`.job.annotation` etc.). could result in several single-item objects.

With that said, I'm fine with any approach. And realistically this certgen component isn't going to grow much. 😊 

Let me know what you think.

Reply:
makes sense to me, agreed on the single level keys not being the best solution.